book/src/volume-cloning.md

```diff
@@ -0,0 +1,46 @@
+# Volume Cloning
+
+## Status and Releases
+
+** Status:** Alpha
+
+Status | Min k8s Version | Max k8s version | external-provisioner Version
+--|--|--|--
+Alpha | 1.15 | - | 1.3
+
+## Overview
+
+A Clone is defined as a duplicate of an existing Kubernetes Volume.  For more information on cloning in Kubernetes see the concepts doc for [Volume Cloning](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#volume-cloning).  A storage provider that allows volume cloning as a create feature, may choose to implement volume cloning via a control-plan CSI RPC call.
+
+For details regarding the kuberentes API for volume cloning, please see [kubernetes concepts](https://kubernetes.io/docs/concepts/storage/volume-pvc-datasource/).
+
+## Implementing Volume cloning functionality
+
+To implement volume cloning the CSI driver MUST:
+
+1. Implement checks for `csi.CreateVolumeRequest.VolumeContentSource` in the plugin's `CreateVolume` function implementation.
+2. Implement `CLONE_VOLUME` controller capability.
+
+It is the responsibility of the storage plugin to either implement an expansion after clone if a provision request size is greater than the source, or to allow the external-resizer handle it.  In the case that the plugin does not support resize capability and it does not have the capability to create a clone that is great in size than the specified source volume, then the provision request should result in a failure.
```

Comment:
typo: greater

Reply:
?

Reply:
great => greater

Reply:
Ah, sorry I was stuck at "expansion after clone if a provision request size is greater than the"